src/browser/modules/D3Visualization/components/Explorer.tsx

```diff
@@ -174,73 +193,44 @@ export class ExplorerComponent extends Component<any, ExplorerComponentState> {
     // and also doing this in a different order from the graph. This leads to different default colors being assigned to different labels.
-    let legend
-    if (this.state.freezeLegend) {
-      legend = (
-        <LegendComponent
-          stats={this.state.stats}
-          graphStyle={neoGraphStyle()}
-          onSelectedLabel={this.onSelectedLabel.bind(this)}
-          onSelectedRelType={this.onSelectedRelType.bind(this)}
-        />
-      )
-    } else {
-      legend = (
-        <LegendComponent
-          stats={this.state.stats}
-          graphStyle={this.state.graphStyle}
-          onSelectedLabel={this.onSelectedLabel.bind(this)}
-          onSelectedRelType={this.onSelectedRelType.bind(this)}
-        />
-      )
-    }
-    const inspectingItemType =
-      !this.state.inspectorContracted &&
-      ((this.state.hoveredItem && this.state.hoveredItem.type !== 'canvas') ||
-        (this.state.selectedItem && this.state.selectedItem.type !== 'canvas'))
+    const graphStyle = this.state.freezeLegend
+      ? neoGraphStyle()
+      : this.state.graphStyle
 
     return (
-      <StyledFullSizeContainer
-        id="svg-vis"
-        className={
-          Object.keys(this.state.stats.relTypes).length ? '' : 'one-legend-row'
-        }
-      >
-        {legend}
-        <GraphComponent
-          fullscreen={this.props.fullscreen}
-          frameHeight={this.props.frameHeight}
-          relationships={this.state.relationships}
-          nodes={this.state.nodes}
-          getNodeNeighbours={this.getNodeNeighbours.bind(this)}
-          onItemMouseOver={this.onItemMouseOver.bind(this)}
-          onItemSelect={this.onItemSelect.bind(this)}
-          graphStyle={this.state.graphStyle}
-          styleVersion={this.state.styleVersion} // cheap way for child to check style updates
-          onGraphModelChange={this.onGraphModelChange.bind(this)}
-          assignVisElement={this.props.assignVisElement}
-          getAutoCompleteCallback={this.props.getAutoCompleteCallback}
-          setGraph={this.props.setGraph}
-        />
-        <InspectorComponent
-          hasTruncatedFields={this.props.hasTruncatedFields}
-          fullscreen={this.props.fullscreen}
-          hoveredItem={this.state.hoveredItem}
-          selectedItem={this.state.selectedItem}
-          graphStyle={this.state.graphStyle}
-          onExpandToggled={this.onInspectorExpandToggled.bind(this)}
-        />
+      <StyledFullSizeContainer id="svg-vis">
+        <StyledGraphAreaContainer>
+          <GraphComponent
+            fullscreen={this.props.fullscreen}
+            frameHeight={this.props.frameHeight}
+            relationships={this.state.relationships}
+            nodes={this.state.nodes}
+            getNodeNeighbours={this.getNodeNeighbours.bind(this)}
+            onItemMouseOver={this.onItemMouseOver.bind(this)}
+            onItemSelect={this.onItemSelect.bind(this)}
+            graphStyle={this.state.graphStyle}
```

Comment:
should this not be `graphStyle` using the above variable like in the props for `<NodeInspectorPanel>` ?